explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -329,26 +306,35 @@ object TargetTabContents {
     }
   }
 
-  protected class Backend($ : BackendScope[Props, State]) {
-    def render(props: Props) = {
-      implicit val ctx = props.ctx
-      AsterismGroupLiveQuery(
-        Reuse(renderFn _)(props, ViewF.fromState($))
-      )
-    }
-  }
-
   protected val component =
-    ScalaComponent
-      .builder[Props]
-      .initialState(
-        State(TwoPanelState.initial[ObsIdSet](SelectedPanel.Uninitialized),
-              TargetVisualOptions.Default
+    ScalaFnComponent
+      .withHooks[Props]
+      .useStateView(TwoPanelState.initial[ObsIdSet](SelectedPanel.Uninitialized))
+      .useStateView(TargetVisualOptions.Default)
+      .useEffectOnMountBy { (props, panels, _) =>
+        implicit val ctx = props.ctx
+        Callback.log("On mount") *>
```

Comment:
A bit of nitpicking: since we already have an implicit `AppContextIO` in context, this could be for example `Logger[IO].debug` or `.trace`, so that we can control the log level.

Reply:
I'll remove this, it is a left over while I was developing